pkg/apihandler/apihandler.go

```
@@ -1778,23 +1857,31 @@ func (f *httpFlushWriter) Write(p []byte) (n int, err error) {
 			return 0, err
 		}
 	}
-	if f.sse {
-		if f.sseBuf == nil {
-			f.sseBuf = &bytes.Buffer{}
-		}
-		if f.sseBuf.Len() == 0 {
-			f.sseBuf.WriteString("data: ")
-		}
-		return f.sseBuf.Write(p)
-	}
-	return f.writer.Write(p)
+
+	return f.buf.Write(p)
 }
 
 func (f *httpFlushWriter) Flush() {
-	if f.sse && f.sseBuf != nil {
-		_, _ = f.sseBuf.WriteTo(f.writer)
-		f.sseBuf.Reset()
+	resp := f.buf.Bytes()
+	f.buf.Reset()
+
+	if f.postResolveCallback != nil {
+		(*f.postResolveCallback)(f.ctx, resp)
+	}
+
+	if f.mutatingPostResolveCallback != nil {
+		if r, err := (*f.mutatingPostResolveCallback)(f.ctx, resp); err == nil {
+			resp = r
```

error gets swallowed

---

they are logged

---

ok, I didn't see.